index.html

```diff
@@ -61,14 +61,138 @@ <h4 class="heading">Skills</h4>
+        <div class="shining-star"></div>
+        <div class="grid-row-item">
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
+        </div>
+        <div class="grid-row-item">
+          <button type="button" class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+      <div class="grid-item proj-2">
+        <div class="grid-row-item">
+          <img src="images/Img-empty.png" alt="Project Image" class="project-image"/>
+        </div>
+        <div class="grid-row-item">
+          <h4 class="project-title">Multi-Post Stories</h4>
+        </div>
+        <div class="shining-star"></div>
+        <div class="grid-row-item">
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
+        </div>
+        <div class="grid-row-item">
+          <button type="button" class="btn">
+            See Project
+          </button>
+        </div>
+      </div>
+      <div class="grid-item proj-3">
+        <div class="grid-row-item">
+          <img src="images/Img-empty.png" alt="Project Image" class="project-image"/>
+        </div>
+        <div class="grid-row-item">
+          <h4 class="project-title">Multi-Post Stories</h4>
+        </div>
+        <div class="shining-star"></div>
+        <div class="grid-row-item">
+          <p class="project-description">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+        </div>
+        <div class="grid-row-item">
+          <ul class="lang-list">
+            <li class="listitems">JavaScript</li>
+            <li class="listitems">C#</li>
+            <li class="listitems">C++</li>
+            <li class="listitems">HTML</li>
+            <li class="listitems">CSS</li>
+          </ul>
```

Comment:
- [ ] Kindly ensure you centralize your list of technologies with the appropriate margin as specified in the Figma template you chose👍🏼 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/185468739-c40603a7-8331-439d-bd55-41cf94373e86.png)  | ![image](https://user-images.githubusercontent.com/63473584/185468894-0c95ec33-a3c9-46ce-bb2e-4f9bd6641fe5.png)  |